erc20/vyper/Makefile

```diff
@@ -1,13 +1,12 @@
 BUILD_DIR:=../../.build
 
-LOCAL_LEMMAS:=../abstract-semantics.k ../verification.k
+LOCAL_LEMMAS:=../abstract-semantics.k ../verification.k ../../resources/evm-symbolic.k
 TMPLS:=../module-tmpl.k ../spec-tmpl.k
 
 SPEC_GROUP:=vyper-erc20
 SPEC_INI:=vyper-erc20-spec.ini
 
-KPROVE_OPTS:=--smt-prelude $(abspath $(dir $(abspath $(lastword $(MAKEFILE_LIST))))/../../resources/evm.smt2) \
-             --deterministic-functions --cache-func-optimized
+KPROVE_OPTS:=--smt-prelude $(abspath $(dir $(abspath $(lastword $(MAKEFILE_LIST))))/../../resources/evm.smt2)
 
```

Comment:
I believe we can wait until I fix K-gnosis-rebased, to avoid messing up the scripts. Whoever depends on this PR can rebase on it.

Reply:
sure